Vision/AnalyzeFormV2/AnalyzeInvoice/__init__.py

```diff
@@ -0,0 +1,152 @@
+import logging
+import json
+import os
+import logging
+import datetime
+from json import JSONEncoder
+
+import azure.functions as func
+from azure.ai.formrecognizer import FormRecognizerClient
+from azure.ai.formrecognizer import FormTrainingClient
+from azure.core.credentials import AzureKeyCredential
+
+
+class DateTimeEncoder(JSONEncoder):
+        #Override the default method
+        def default(self, obj):
+            if isinstance(obj, (datetime.date, datetime.datetime)):
+                return obj.isoformat()
+
+
+def main(req: func.HttpRequest) -> func.HttpResponse:
+    logging.info('Python HTTP trigger function processed a request.')
+
+    try:
+        body = json.dumps(req.get_json())
+        if body:
+            logging.info(body)
+            result = compose_response(body)
+            return func.HttpResponse(result, mimetype="application/json")
+        else:
+            return func.HttpResponse(
+                "Invalid body",
+                status_code=400
+            )
+    except ValueError:
+        return func.HttpResponse(
+             "Invalid body",
+             status_code=400
+        )
+    
+    
+
+
+def compose_response(json_data):
+    values = json.loads(json_data)['values']
+    
+    # Prepare the Output before the loop
+    results = {}
+    results["values"] = []
+    endpoint = os.environ["FORMS_RECOGNIZER_ENDPOINT"]
+    key = os.environ["FORMS_RECOGNIZER_KEY"]
+    form_recognizer_client = FormRecognizerClient(endpoint, AzureKeyCredential(key))
+    for value in values:
+        output_record = transform_value(value, form_recognizer_client)
+        if output_record != None:
+            results["values"].append(output_record)
+    return json.dumps(results, ensure_ascii=False, cls=DateTimeEncoder)
```

Comment:
Should the boilerplate code be moved into some kind of utility class like we did in C#?